examples/assemblyscript/main/smartlamp.ts

```diff
@@ -40,7 +40,7 @@ export function main(): void {
     // Connect to Wi-Fi
     until_connected(
         () => { wd.wifi_connect(SSID, PASSWORD); },
-        () => { return wd.wifi_status() == wd.WL_CONNECTED; });
+        wd.wifi_connected);
     let message = "Connected to wifi network with ip: ";
```

Comment:
@tolauwae , will this work?

Reply:
It should. I will test on an ESP tomorrow.